pom.xml

```diff
@@ -105,6 +105,11 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
+            <groupId>org.eclipse.aether</groupId>
```

Comment:
I'd like some clarification on this, but yeah -- nice catch on why it wasn't working. 👍